.travis.yml

```diff
@@ -14,4 +26,11 @@ before_cache:
 
 notifications:
   email:
-    on_success: never
\ No newline at end of file
+    on_success: never
```

Comment:
Are we sure we want this? it's a useful notification for me sometimes :). But this is unrelated to this PR

Reply:
Yea, I also find these emails useful. I do not disable notifications when creating new `travis.yml` file. But since it was here already, I did not dare to remove it. :)